packages/saber/lib/webpack/webpack.config.js

```diff
@@ -94,6 +94,10 @@ module.exports = (api, { type }) => {
     }
   ])
 
+  config
```

Comment:
@egoist should I check for `build.cache`

Reply:
yes

Reply:
done